pymessenger/user_profile.py

```diff
@@ -1,17 +1,17 @@
 import requests
 
 from pymessenger.graph_api import FacebookGraphApi
-    
+
 class UserProfileApi(FacebookGraphApi):
-    def get(self, user_id, fields=None):
+    def get(self, user_id, fields=None, request_kwargs={}):
```

Comment:
Can you set this in the FacebookGraphApi __init__ method, then use it in requests anywhere they are made?